app/src/main/java/at/aau/edu/wizards/MainActivity.kt

```diff
@@ -15,11 +18,26 @@ class MainActivity : AppCompatActivity() {
                 showSampleFragment()
             }
         }
+
+        handlePermissions()
     }
 
     private fun showSampleFragment() {
         supportFragmentManager.beginTransaction()
             .add(R.id.fragment_container, SampleFragment(), "TAG")
             .commit()
     }
+
+    private fun handlePermissions() {
+        val permissionHandler = PermissionHandler(
+            activity = this,
+            permissions = REQUIRED_PERMISSIONS
+        ) { permissions ->
+            val anyDenied = permissions.any { !it.value }
+
+            println("Any permissions denied = $anyDenied")
+            // TODO: Need to properly handle denied permissions. App won't work without
```

Comment:
Same question about what to do with todos. (*badumz*) Create backlog tasks?

Reply:
yadayada

Reply:
fairs